code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -1226,6 +1226,17 @@
 		RND_CATEGORY_MECHFAB_CYBORG_MODULES + RND_SUBCATEGORY_MECHFAB_CYBORG_MODULES_JANITOR
 	)
 
+/datum/design/borg_upgrade_uwu
+	name = "Cyborg UwU-speak 'Upgrade'"
+	id = "borg_upgrade_cringe"
+	build_type = MECHFAB
+	build_path = /obj/item/borg/upgrade/uwu
+	materials = list(/datum/material/gold = 2000, /datum/material/diamond = 1000, /datum/material/bluespace = 500)
+	construction_time = 120
```

Comment:
Should be 12 SECONDS

Reply:
I followed the pattern of similar parts, but I think that would make it much clearer. Some of these units of measurement in tg are _weird_, man.